src/transformers/__testfixtures__/import-declaration-transform.output.js

```diff
@@ -4,6 +4,8 @@ import foo from './new/path/to/foo';
 export * from './new/path/to/bar';
 export {default as bar} from './new/path/to/bar';
 export {default as foo} from './new/path/to/foo';
+export const num = 1;
+export const filePath = './new/path/to/bar';
```

Comment:
Also, we should not be updating the path here since it's a string literal and not a source node. Please update to:

```js
export const filePath = './bar';
```